engine-nextgen/src/test/kotlin/graphql/nadel/validation/NadelHydrationValidationTest.kt

```diff
@@ -124,6 +127,58 @@ class NadelHydrationValidationTest : DescribeSpec({
             errors.assertSingleOfType<MissingHydrationActorField>()
         }
 
+        it("fails if hydration actor field exists only in the underlying and not in the overall") {
+            val fixture = NadelValidationTestFixture(
+                overallSchema = mapOf(
+                    "issues" to """
+                        type Query {
+                            issue: Issue
+                        }
+                        type Issue {
+                            id: ID!
+                            creator: User @hydrated(
+                                service: "users"
+                                field: "user"
+                                arguments: [
+                                    {name: "id", value: "$source.creator"}
+                                ]
+                            )
+                        }
+                    """.trimIndent(),
+                    "users" to """
+                        type User {
+                            id: ID!
+                            name: String!
+                        }
+                    """.trimIndent(),
+                ),
+                underlyingSchema = mapOf(
+                    "issues" to """
+                        type Query {
+                            issue: Issue
+                        }
+                        type Issue {
+                            id: ID!
+                            creator: ID!
+                        }
+                    """.trimIndent(),
+                    "users" to """
+                        type Query {
+                            user(id: ID!): User
+                        }
+                        type User {
+                            id: ID!
+                            name: String!
+                        }
+                    """.trimIndent(),
+                ),
+            )
+
+            val errors = validate(fixture)
+
+            assert(errors.map { it.message }.isNotEmpty())
```

Comment:
Can we have a more specific assert here? Checking for more specific error messages makes the test more accurate and easier to understand what it is actually testing.

Reply:
wait I swear I just pushed a commit with that

Reply:
@felipe-gdr I think you are looking at the old diff, take a look here https://github.com/atlassian-labs/nadel/pull/369/files#diff-5737c48bda1f8571497df6122df9345befa787904d7fd2f66d3900d507d4a4f7R181